### PR TITLE
37259: changes to address restricted scc

### PR DESCRIPTION
**Pull request by cheewaio**

### CODE REVIEW

File: deploy/olm-catalog/ibm-platform-api-operator/3.6.0/ibm-platform-api-operator.v3.6.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -298,7 +298,7 @@ spec:
                   privileged: false
                   readOnlyRootFilesystem: true
                   runAsNonRoot: true
-                  runAsUser: 1001
+                  runAsUser: 1000550000
```
Comment (bitscuit) [inline deploy/olm-catalog/ibm-platform-api-operator/3.6.0/ibm-platform-api-operator.v3.6.0.clusterserviceversion.yaml:301]: does the user have to be explicitly set? I thought OCP can automatically assign the user

File: deploy/operator.yaml [hunk 2]
```diff
@@ -49,7 +49,7 @@ spec:
             privileged: false
             readOnlyRootFilesystem: true
             runAsNonRoot: true
-            runAsUser: 1001
+            runAsUser: 1000550000
```
Comment (bitscuit) [inline deploy/operator.yaml:52]: same comment about user

Comment (cheewaio) [reply deploy/operator.yaml:52]: That's a question for @bluzarraga, it's mainly to avoid linting error from case bundle travis build

Comment (bluzarraga) [reply deploy/operator.yaml:52]: there will be a linting error if it's not included, so it's a matter of whether we want it explicitly set or if we want to override it and let ocp assign the user

**Pull request merged**